FORGE/DIOGOPINTO/myForge/Assignments/#6/climate_vpd.py

```diff
@@ -16,14 +16,24 @@
 #Create a new csv file with an extra column: VPD Values
 data_vpd= data.copy()
 
+
 data_vpd['VPD Values']= round(mf6.calculate_vpd(Tmax,Tmin,RHmean),4)
 
-data_vpd.to_csv(r'/Users/diogopinto/Documents/VScode/greends-ipython/FORGE/DIOGOPINTO/myForge/Assignments/#6/Climate_hisafe_knmievaluation_daily_from_01-01-1981_to_31-12-2010_vpd.csv')
+
+
+data_vpd.columns
+print(data_vpd)
+
+data_vpd.to_csv(r'/Users/diogopinto/Documents/VScode/greends-ipython-1/FORGE/DIOGOPINTO/myForge/Assignments/#6/Climate_hisafe_knmievaluation_daily_from_01-01-1981_to_31-12-2010_vpd.csv')
```

Comment:
@24832 Use relative paths?
just save it in the assignment folder

Reply:
data_vpd.to_csv(r'Climate_hisafe_knmievaluation_daily_from_01-01-1981_to_31-12-2010_vpd.csv')